src/Angor/Client/Components/Icon.razor

```diff
@@ -0,0 +1,22 @@
+﻿@inject Angor.Client.Services.IconService IconService
+@code {
+    [Parameter] public string IconName { get; set; }
+    [Parameter] public int Width { get; set; } = 32;
+    [Parameter] public int Height { get; set; } = 32;
+    [Parameter] public string Color { get; set; } = "var(--bs-primary)";
+    private string svgContent;
+
+    protected override async Task OnParametersSetAsync()
+    {
+        svgContent = await IconService.GetIcon(IconName, Width, Height, Color);
```

Comment:
Its not cached so does that not mean the icon is fetched every load?

Reply:
The icons are placed in the application that are downloaded in the user's browser, and their loading does not mean downloading from the server.